src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import '../../App.css';
+import CarouselText from './CarouselText';
+import carousel_img_1 from './images/carousel_img_1.png';
+import carousel_img_2 from './images/carousel_img_2.png';
+import carousel_img_3 from './images/carousel_img_3.png';
+
+function Carousel() {
```

Comment:
function starts with small letter, only when you import it in component you name it witch capital letter

Reply:
Changed

Reply:
I import Carousel to the App